app/enterprise/0.33-x/developer-portal/best-practice.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Kong Developer Portal Best Practices
+book: portal
+chapter: 12
+---
+
+# Kong Developer Portal Best Practices
+
+The Kong Developer Portal is a powerful tool. It's part static website and part interactive API. With  tool of this magnitude, there are countless ways to configure and host that will vary greatly depending on your needs. The following article is a group of recommendations we have found to lead which lead to  easy administrative and top performance. This list is not all inclusive, so if you have a trick you like to use and want to share it with others, please feel free to make a PR on the page, or send us an email at <support@konghq.com> and we can add it for you.
+
+___
+
+- [Basic System Requirements and Initial Configuration](#systemConfig)
+- [Customizing The Portal Design with CSS](#css)
+- [Customizing Front End Behavior with JavaScript](#js)
+- [Using Source Control](#git)
+- [Effective Partial Usage](#partial)
+- [Practices to Avoid](#avoid)
+___
+
+## <a name=systemConfig></a>Basic System Requirements and Initial Configuration
+
+When deciding what type of system to host Kong on, you should take into consideration into the amount of traffic you expect the portal to see. If you are using your portal only for internal APIs shared between 2 teams, you'll need a much smaller instance than someone with a public facing, heavily trafficked API. By default, I suggest starting with a more powerful machine than you need and scaling back. if you have a bit more power than you need, none of your end users will notice a difference. However, if you start too small, it could potentially lead to a slower performance of the portal. This section is going to assume your portal gets about 1000 concurrent visitors on average.
+
+If you are using AWS for you hosting, I would provision a [t2.large](https://aws.amazon.com/ec2/instance-types/t2/) as a starting place. This is a solid base and should be able to handle our expectations without any issue. If you use Google Cloud (GCE), I would suggest starting with a [n1-standard-2](https://cloud.google.com/compute/pricing#standard_machine_types) and if you are using the Azure cloud, I would suggest starting with a [Standard_D2_v3](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-sup1sup).
+
+If you look at those recommendations, you'll see they all have similar starts:
+  - At least 2 CPUs
+  - Around 8 GiB System memory
+
+I also recommend having this Kong instance be used to only serve the dev portal, and not do any Kong administrative work. To do this, in your kong.conf file, set the following values
+```
+admin_listen=off
+portal=on
+```
+We will also want to ensure all other instances of Kong have the portal boolean set to off. This ensures your portal is only being served by a single Kong instance.
```

Comment:
Please link to the other doc in which these configuration options are explained.

Reply:
Link "portal boolean set to off" to https://docs.konghq.com/enterprise/0.33-x/developer-portal/configuration/getting-started/